.claude/hooks/post_tool_use_python_check.py

```
@@ -111,66 +107,187 @@ def run_mypy(file_path):
+
+    except subprocess.TimeoutExpired:
+        errors.append(f"LanguageTool timed out for {file_path}")
+    except Exception:
+        # Silently skip LanguageTool errors to avoid breaking the hook
+        pass
+
     return errors
 
-def main():
+
+def main() -> None:
```

Ruff (0.12.2)

`main` is too complex (15 > 10)

(C901)